packages/icons/src/warning-icon.svg

```diff
@@ -1 +1 @@
-<svg viewBox="0 0 32 32" fill="none" xmlns="http://www.w3.org/2000/svg"><path fill-rule="evenodd" clip-rule="evenodd" d="M16 28.988c-7.179 0-13-5.82-13-13s5.821-13 13-13c7.18 0 13 5.82 13 13s-5.82 13-13 13Zm1.501-6.493a1.5 1.5 0 1 0-3 0 1.5 1.5 0 0 0 3 0Zm-1.5-3.517a1 1 0 0 0 1-1V9.026a1 1 0 1 0-2 0v8.952a1 1 0 0 0 1 1Z" fill="#FF7B26"/><path fill-rule="evenodd" clip-rule="evenodd" d="M16 27.988c-6.627 0-12-5.373-12-12 0-6.628 5.373-12 12-12 6.628 0 12 5.372 12 12 0 6.627-5.372 12-12 12Zm2.501-5.493a2.5 2.5 0 1 0-5 0 2.5 2.5 0 0 0 5 0Zm-2.5-2.517a2 2 0 0 0 2-2V9.026a2 2 0 1 0-4 0v8.952a2 2 0 0 0 2 2Zm-13-3.99c0 7.18 5.82 13 13 13s13-5.82 13-13-5.82-13-13-13-13 5.82-13 13Zm13 5.007a1.5 1.5 0 1 1 0 3 1.5 1.5 0 0 1 0-3Zm1-3.017a1 1 0 1 1-2 0V9.026a1 1 0 1 1 2 0v8.952Z" fill="#FF7B26"/></svg>
\ No newline at end of file
+<svg viewBox="0 0 32 32" fill="currentColor"  xmlns="http://www.w3.org/2000/svg"><path fill-rule="evenodd" clip-rule="evenodd" d="M16 28.988c-7.179 0-13-5.82-13-13s5.821-13 13-13c7.18 0 13 5.82 13 13s-5.82 13-13 13Zm1.501-6.493a1.5 1.5 0 1 0-3 0 1.5 1.5 0 0 0 3 0Zm-1.5-3.517a1 1 0 0 0 1-1V9.026a1 1 0 1 0-2 0v8.952a1 1 0 0 0 1 1Z" fill="#FF7B26"/><path fill-rule="evenodd" clip-rule="evenodd" d="M16 27.988c-6.627 0-12-5.373-12-12 0-6.628 5.373-12 12-12 6.628 0 12 5.372 12 12 0 6.627-5.372 12-12 12Zm2.501-5.493a2.5 2.5 0 1 0-5 0 2.5 2.5 0 0 0 5 0Zm-2.5-2.517a2 2 0 0 0 2-2V9.026a2 2 0 1 0-4 0v8.952a2 2 0 0 0 2 2Zm-13-3.99c0 7.18 5.82 13 13 13s13-5.82 13-13-5.82-13-13-13-13 5.82-13 13Zm13 5.007a1.5 1.5 0 1 1 0 3 1.5 1.5 0 0 1 0-3Zm1-3.017a1 1 0 1 1-2 0V9.026a1 1 0 1 1 2 0v8.952Z" fill="#FF7B26"/></svg>
```

Comment:
there's still hardcoded hex colors in the path elements

Reply:
done.